docs/cypress/e2e/links.cy.ts

```diff
@@ -0,0 +1,137 @@
+import { VALIDATED_LINKS } from '../data/validatedLinks';
+import { REQUEST_GET_LINKS } from '../data/requestGetLinks';
+import { BASE_URL } from '../data/constants';
+
+let allLinks: string[] = [];
+const numberOfLinks = 119;
+const requestedLinks: Set<string> = new Set();
+
+before(() => {
+  cy.task('readSitemapLinks').then((links: string[]) => {
+    allLinks = allLinks.concat(links);
+  });
+});
+
+describe('Local Sitemap', () => {
+  it('should have 119 links', () => {
+    expect(allLinks.length).to.eq(numberOfLinks);
+  });
+});
+
+for (let i = 0; i < numberOfLinks; i++) {
+  describe(`check page ${i}`, () => {
```

Comment:
Nit: should we move this describe outside of the `for` loop? Seems odd to have 119 describe blocks each with only 1 test, instead of 1 describe with 119 test.

Reply:
Good point. It looks better now.


![image](https://user-images.githubusercontent.com/11983489/198711186-cf8c7940-131d-4019-a3f9-52a2230f47c8.png)